check-update-security.sh

```diff
@@ -153,7 +153,7 @@ check_wait_response $(echo $?)
 # Instance has been launched, terminate in case of an error
 trap 'failure_cleanup' ERR
 
-rm "$user_data"
+rm user_data.txt
```

Comment:
because we went from mktemp to touch as the action, touch doesn't return the name of the file, just a success code. Therefore, we can't try to remove "0" or anything like that. We have to actually remove the file.